src/modal/_pera-wallet-modal.scss

```
@@ -23,6 +23,13 @@
 
   * {
     box-sizing: border-box;
+
+    li {
+      list-style-type: none;
+
+      margin: 0;
+      padding: 0;
+    }
   }
```

Why is it nested under * ? 
```suggestion
  * {
    box-sizing: border-box;
  }
  
    li {
      list-style-type: none;

      margin: 0;
      padding: 0;
    }
```